src/Sulu/Component/Content/Compat/Structure/StructureBridge.php

```diff
@@ -322,7 +322,8 @@ public function toArray($complete = true)
             'nodeType' => $this->getNodeType(),
             'nodeState' => $this->getNodeState(),
             'internal' => false,
-            'contentLocales' => $this->inspector->getLocales($document),
+            'availableLocales' => $this->inspector->getLocales($document),
```

Comment:
whats the reason for this change?

Reply:
Because the value of these properties were different when you were requesting multiple pages vs a single page. Adjusted it to be consistent.